array_api_tests/test_creation_functions.py

```diff
@@ -130,9 +150,31 @@ def test_full(shape, fill_value, dtype):
     else:
         assert all(equal(a, asarray(fill_value, **kwargs))), "full() array did not equal the fill value"
 
-# TODO: implement full_like (requires hypothesis arrays support)
-def test_full_like():
-    pass
+@given(
+    a=xps.arrays(
+        dtype=shared_dtypes,
+        shape=xps.array_shapes(),
+    ),
+    fill_value=shared_dtypes.flatmap(promotable_dtypes).flatmap(xps.from_dtype),
```

Comment:
Ah, I didn't see that `from_dtype` was added. That is useful, and can probably replace some of the strategies in the hypothesis helpers (assuming it is implemented the same way).